plugin/src/main/kotlin/nl/littlerobots/vcu/plugin/VersionCatalogFormatTask.kt

```diff
@@ -47,6 +47,11 @@ abstract class VersionCatalogFormatTask @Inject constructor() : DefaultTask() {
         keep.orNull?.getVersionCatalogRefs() ?: emptySet()
     }
 
+    init {
+        description = "Formats the Gradle versions catalog file."
```

Comment:
Should this also be `Formats the libs.version.toml file` to be consistent with the other descriptions? Otherwise the correct name is `version catalog` according to the Gradle docs.

Reply:
I went with explicitly mentioning `libs.versions.toml`.